dts/bindings/usb/nxp,uhc-ehci.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright  2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP USB host controller
```

Comment:
Can we add the IP name to these descriptions, so something like `NXP EHCI USB host controller`?